app/views/components/_published-dates.html.erb

```diff
@@ -0,0 +1,30 @@
+<%
+  published ||= false
+  history ||= false
+  last_updated ||= false
+  link_to_history ||= false
+%>
+
+<div class="app-c-published-dates" <% if history%>id="history" data-module="toggle"<% end %>>
```

Comment:
Should the block be wrapped in an `if published || last_updated` so that if nothing is passed in nothing renders?